### ISSUE TITLE
Unable to set deadline for task even though valid date is provided

### ISSUE THREAD

**Issue by czhongwei** [czhongwei/pe#2]
Prerequisite: have a task in the task list already

Command given: task do 2 by/20 December 2022

Expected: task at index 2 given a deadline of 20 December 2022

Actual: Error message saying provided date could not be parsed
![Screenshot 2022-11-11 at 4.32.05 PM.png](https://raw.githubusercontent.com/czhongwei/pe/main/files/2b32fd86-9951-4500-b022-a7016f073e4f.png)
Even commands that are similar do not work, e.g "task do 2 by/20 Dec 2022", "task do 2 by/20 December"
However, another command in a similar date format but different date works- "task do 2 by/19 December 2022"
![Screenshot 2022-11-11 at 4.33.28 PM.png](https://raw.githubusercontent.com/czhongwei/pe/main/files/e542413b-5a35-4989-9298-7f938cd79e80.png)

Severity given a medium because with several other inputs, such as 20 November 2023 and 20 August 2023, problem still persists. It is not uncommon to set such dates as deadlines. Seems like dates above 19th of any month will cause this error.

**Comment by nus-se-script** [comment]
# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Setting deadlines not accepting valid dates
>
>## Steps to reproduce
>1. Run the command `task do 1 by/20 February 2022` (19 February 2022 works)
>
>## Expected
>1. As it is a valid date, the deadline of the task should be changed accordingly.
>
>
>## Actual
>1. A non-specific error message is thrown
>
>## Screenshot / Video
>
>![Screenshot 2022-11-11 at 4.49.30 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2223S1/pe/master/files/57f5d2ec-8ff1-4ce8-9263-94e2f2950f02.png)
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2223S1/pe-interim#1238] [original labels: severity.Medium type.FunctionalityBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>No details provided by team.
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

-------------------
